README.md

```diff
@@ -1,20 +1,50 @@
 # Valkey Admin
 
-## Getting Started (Development)
+## What is Valkey Admin?
 
-### Start Valkey Cluster
+Valkey Admin is a web-based administration tool for Valkey clusters. It provides an intuitive interface to monitor, manage, and interact with your Valkey instances, offering features like real-time metrics, key management, and cluster administration.
 
-1. From the root directory, run `tools/valkey-cluster/scripts/build_run_cluster.sh`.
-    1. This will build and run the cluster.
-    2. Master hostname will be `localhost`.
-    3. Master nodes ports will be `7001` - `7003`.
+Built with React and TypeScript, Valkey Admin runs as a desktop application (via Electron) or web application, making it easy to manage your Valkey infrastructure from anywhere.
```

Comment:
Some features like hotkeys and commandlogs rely on Electron, so currently we only support it as a desktop app.